zanata-war/src/main/java/org/zanata/events/webhook/DocumentMilestoneEvent.java

```diff
@@ -18,7 +18,7 @@
  * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
  * site: http://www.fsf.org.
  */
-package org.zanata.events;
+package org.zanata.events.webhook;
```

Comment:
I think we should keep the webhook payloads in a completely separate package from our CDI events `org.zanata.events`. Maybe `org.zanata.webhook.events`?